internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
```

Comment:
Review the context usage. If the context is intended to control the lifecycle of requests, consider passing it as part of the request context instead of using a general context passed at the start, to ensure that it is appropriately scoped.